lib/internal/modules/esm/loader.js

```diff
@@ -89,10 +136,20 @@ class DefaultModuleLoader {
    */
   loaderType = 'default';
 
-  constructor() {
+  /**
+   * Loader to pass requests to.
+   */
+  #delegate = null;
+
+  constructor(delegate = null) {
```

Comment:
Generally a default value of `undefined` is preferable.

```suggestion
  constructor(delegate) {
```